tomb/benches/pipeline.rs

```diff
@@ -23,10 +27,10 @@ lazy_static! {
     static ref BENCH_PATH: String = env::var("BENCH_PATH").unwrap_or_else(|_| "target/bench".to_string());
     // Path we will use to hold inputs we build for benchmarking
     static ref INPUT_PATH: String = env::var("INPUT_PATH").unwrap_or_else(|_| "target/bench/input".to_string());
-    // Path we will use to hold packed data we build from inputs for benchmarking
-    static ref PACKED_PATH: String = env::var("PACKED_PATH").unwrap_or_else(|_| "target/bench/packed".to_string());
-    // Path we will use to hold unpacked data we build from packed data for benchmarking
-    static ref UNPACKED_PATH: String = env::var("UNPACKED_PATH").unwrap_or_else(|_| "target/bench/unpacked".to_string());
+    // Path we will use to hold bundleed data we build from inputs for benchmarking
```

Comment:
There is a typo kind of throughout this file of "bundleed" (with an extra e) instead of bundled. Can't tell if that was intentional or not hahah